scripts/plot_camera_image.py

```diff
@@ -4,79 +4,198 @@
 
 import matplotlib.pyplot as plt
 import numpy as np
-    
+
 plt.rc('xtick', labelsize=22)
 plt.rc('ytick', labelsize=22)
 
-test_mapper = ImageMapper()
-test_mapper_pad = ImageMapper(padding = {
-                                      'LST': 10,
-                                      'MSTF': 10,
-                                      'MSTN': 20,
-                                      'MSTS': 10,
-                                      'SST1': 10,
-                                      'SSTC': 10,
-                                      'SSTA': 10,
-                                      'VTS': 10,
-                                      'MGC': 5,
-                                      'FACT': 10,
-                                      'HESS-I': 10,
-                                      'HESS-II': 10
-                                      })
-    
-test_im_dict = {}
-test_im_dict_pad = {}
+test_mapper_over = ImageMapper(hex_conversion_algorithm='oversampling')
```

Comment:
All of the options here need to be updated to match the new argument style. Hex_conversion_algorithm should be a dict. The camera type names need to be updated as well.